drivers/volume/mock/mock.go

```diff
@@ -15,9 +15,10 @@ import (
 )
 
 const (
-	driverName       = "MockDriver"
-	storageClassName = "mockDriverStorageClass"
-	provisionerName  = "kubernetes.io/mock-volume"
+	driverName = "MockDriver"
+	// MockDriverStorageClassName is the storage class for mock driver PVCs
+	MockDriverStorageClassName = "mockDriverStorageClass"
```

Comment:
Doesn't need to be exposed?

Reply:
Also, it's already in the mock driver, don't need to add MockDriver to the name

Reply:
fixed